packages/web-components/.npmignore

```diff
@@ -1,30 +1,42 @@
+.vscode/
+
 # Tests
-dist/dts/__test__/
-dist/esm/__test__/
 *.spec.*
+*.test.*
+*.bench.*
 coverage/
+__fixtures__
+__mocks__
+__tests__
 
 # images
 images/
 
 # Source files
 src/
+scripts/
 
 # config files
 .eslintignore
 .eslintrc.js
+.eslintrc.json
 .prettierignore
-.storybook
 tsconfig.json
-tsconfig.build.json
+tsconfig.*.json
 rollup.config.json
+rollup.config.js
 karma.conf.cjs
 api-extractor.json
 .mocharc.json
 
-# Storybook static site
+# Storybook
+.storybook
+*.stories.*
+public/
 dist/storybook-static/
 
 # cache
 .rollupcache
-temp
+temp/
+etc/
+CHANGELOG.json
```

Comment:
follow up - we dont use npmignore in new libraries because they are confusing in git/node resolution and caused issues in the past when migrating to new nodejs versions.

this needs to be refactored to use package.json#files

Reply:
Will resolve after the merge - non blocking per our offline convo. Thanks!